draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx

```diff
@@ -2,6 +2,12 @@ import { MenuContent } from "@kaizen/draft-menu"
 import { StatelessMenu } from "@kaizen/draft-menu/KaizenDraft/Menu/StatelessMenu"
 import classnames from "classnames"
 import * as React from "react"
+/**
+ * Eslint throws a false positive for modules that use require. Ensure you
```

Comment:
I'm a bit confused by the comment - should it be "Eslint throws a false **negative**"? As in, the rule thows an error, but it shouldn't?

"before turning this rule off.." Isn't that what `// eslint-disable-next-line` does - it's already turned off?

Reply:
I think I might just be confused by the semantics of positive and negative 😆

Reply:
positive as in in good, or positive as in this is "positively bad" or "this positively fails the check"?

Reply:
maybe it could read "throws a false error" (if that's what it means)

Reply:
D'oh, you're right. It should read `false negative`, as in, there is an error reported when there shouldn't be. Good catch, i've now fixed this up.